pkg/controller/kubelet-config/kubelet_config_controller_test.go

```diff
@@ -814,3 +816,44 @@ func TestCleanUpDuplicatedMC(t *testing.T) {
 	_, ok = actual[machineConfigUpgrade.Name]
 	require.True(t, ok, "expect generated-kubelet-1 in the list, but got false")
 }
+
+func TestRunKubeletBootstrap(t *testing.T) {
```

Comment:
is a unit test sufficient for this? seems like node should think about creating and maintaining it's own e2e (moving the current e2e out of gcp-op as well) if we want to know /ensure bootstrapping works in the long term?

cc: @rphillips